src/components/content/credential/AddCredential.tsx

```diff
@@ -81,7 +81,19 @@ function AddCredential({
 
     const credentialCapabilitiesQuery = useQuery({
         queryKey: ['credentialCapabilitiesQuery', currentCsp, currentType],
-        queryFn: () => CredentialsManagementService.getCredentialCapabilities(currentCsp, currentType),
+        queryFn: () =>
+            CredentialsManagementService.getCredentialCapabilities(
+                currentCsp as
```

Comment:
we must avoid declaring the CSP values again. We must find any other way of handling the undefined CSP here.